src/transitions/Slide.js

```diff
@@ -17,11 +17,10 @@ function getTranslateValue(props, element) {
     return `translate3d(${0 - (rect.left + rect.width)}px, 0, 0)`;
   } else if (direction === 'up') {
     return `translate3d(0, calc(100vh - ${rect.top}px), 0)`;
-  } else if (direction === 'down') {
-    return `translate3d(0, ${0 - (rect.top + rect.height)}px, 0)`;
   }
 
-  return 'translate3d(0, 0, 0)';
+  // direction === 'down'
```

Comment:
I have cherry-picked that changed into #6911, good catch 👍 !
But I'm closing this PR as has been inactive for quite some time. The test is inconsistent across the different browser. I'm not sure what's going one.